sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/update-noa-status.ts

```diff
@@ -0,0 +1,27 @@
+import { createFakeWorkerJob } from "../../../../../test/utils/worker-job-mock";
+import { IOutputVariables, ZeebeJob } from "zeebe-node";
+import {
+  UpdateNOAStatusHeaderDTO,
+  UpdateNOAStatusJobInDTO,
+} from "../../assessment.dto";
+import { AssessmentStatus } from "@sims/sims-db";
+
+/**
+ * Creates a fake update noa status payload.
```

Comment:
Minor/suggestion/not-a-blocker, but NOA is an acronym that should be always uppercase along all comments.